server.js

```diff
@@ -133,6 +154,6 @@ app.delete('/api/books/:id', function (req, res) {
 
 
 
-app.listen(port, function() {
+app.listen(3500, function() {
```

Comment:
You can use the variable 'port' here because you changed it's value on line 9 above.